components/org.wso2.carbon.metrics.impl/src/test/java/org/wso2/carbon/metrics/impl/ReporterTest.java

```diff
@@ -179,14 +179,14 @@ public void testCSVReporterRestart() {
 
     public void testJDBCReporter() {
         metricService.report();
-        List<Map<String, Object>> meterResult = template.queryForList("SELECT * FROM METRIC_METER WHERE NAME = ?",
-                meterName);
+        List<Map<String, Object>> meterResult =
+                template.queryForList("SELECT * FROM METRIC_METER WHERE NAME = ?", meterName);
         assertEquals("There is one result", 1, meterResult.size());
```

Comment:
@grainier I have a doubt whether this is an *error* message. Is it? It seems the assertion will pass when the meterResult.size() is 1, right?